src/js/lumo.js

```diff
@@ -10,36 +10,108 @@ import JSZip from 'jszip';
 import ArrayStream from './array-stream';
 import * as util from './util';
 
-function inferNodeModulesClasspathLibs(): string[] {
-  const result = [];
+type PackageJsonType = {|
```

Comment:
this is wrong. `{| |}` refers to a closed object, you want `{}`

Reply:
Oh thanks! Sorry not a `flow` master I blindly copied your definitions